scripts/hid_configurator/configurator_cli.py

```diff
@@ -111,6 +112,22 @@ def perform_fwreboot(dev, args):
         print('FW reboot request failed')
 
 
+def perform_led_stream(dev, args):
+    if args.freq <= 0:
+        print('Frequency has to be greater than zero Hz')
+    elif DEVICE[args.device_type]['stream_led_cnt'] == 0:
+        print('Device does not support LED stream')
+    elif args.led_id < 0:
+        print('LED ID cannot be less than zero')
+    elif args.led_id >= DEVICE[args.device_type]['stream_led_cnt']:
```

Comment:
Why do you access global variable here? What more the global is provided by dev argument to the function, correct me if I'm wrong?

Reply:
Yes, you are right. I change id to use dev, thanks for pointing this out.

Reply:
Its not the same. `dev` is value returned by `hid.Device`, `DEVICE` is global containing device configurations.

Reply:
So I suggest to provide device from DEVICE as parameter - do not access global variables and avoid use of them. The code then is not testable.